cms/models/managers.py

```diff
@@ -278,7 +278,7 @@ def subordinate_to_user(self, user, site):
                                         A
                                     /      \
                                   user     B,E
-                                /     \      \
+                                /     \\      \
```

Comment:
This change is probably unwanted... 

```suggestion
                                /     \\      \
                                /     \      \
```